content/en/xlt/load-testing/Manual/570-scorecard.md

```diff
@@ -9,7 +9,210 @@ description: >
+    "testFailTrigger": "NOTPASSED", // rule evaluation status used as trigger when to mark test as failed (has no effect unless "failsTest" is "true") - optional, default = NOTPASSED
+    // the checks to run - optional
+    "checks" : [
+        {
+            "selector": "//requests/request[name[text() = 'Homepage']]/percentiles/p95", //XPath
+            "condition": "<= 1000", //XPath
+            "displayValue" : true, // whether the item matching the selector should be displayed in the report - optional, default = true
+            "enabled": true // helps to disable a check easily, if desired - optional, default = true
+        },
+        {
+            "selectorId": "homepageRuntimeP99", //predefined selector
+            "condition": "<= 3000"
+        }
+    ],
+    // message to display in report when rule does pass/fail - optional, no default value, allowed properties: "success" and "fail"
+    "messages": { 
+        "success" : "C", // use this message in the report when passed - optional, no default value
+        "fail" : "", // otherwise print this text - optional, no default value
+    },
+    "points" : 5 // the number of points to achieve - optional, default = 0, no upper limit
+}
+```
+
+### Checks
+
+Each check is a selector against the XML tree of the testreport.xml using XPath. Rule checks can only have a `selector` or a `selectorId`, not both. Everything selected must be either true or false, a number or a text, or a list of text nodes that later can match a regular expression.
+
+Here are a few examples for checks:
+
+```json
+"checks" : [
+    {
+        // Fetch a number
+        "selector": "//requests/request[name[text() = 'Homepage']]/percentiles/p95",
+        "condition": "<= 1000"
+    },
+    {
+        // Fetch Nodes by Regex
+        "selector": "count(//transactions/transaction/name[matches(text(), 'T.*Cart')])",
+        "condition": "> 0"
+    },
+    {
+        // Compare Against a Number
+        "selector": "count(//agents//totalCpuUsage/mean[number() > 90])",
+        "condition": "= 0"
+    }
+```
+
+### Selectors
+
+It is often helpful to define a set of reusable expressions to query the test result document. These are selectors, which are defined as follows:
+
+```json
+{
+    "selectors" : [
+        {
+            "id": "agentsWithCPULarger60p", // the ID of the selector - required
+            "expression": "count(//agents//totalCpuUsage/mean[number() > 60])", // the XPath expression of the selector - required
+            "comment" : "I can write a note to myself here"  // some comment; won't be evaluated - optional
+        }
+    ]
+}
+```
+
+You can then use the selector in any rule check by referencing its selector id:
+
+```json
+"checks" : [
+    {
+        "selectorId": "agentsWithCPULarger60p", //predefined selector
+        "condition": "= 0"
+    }
+]
+```
```

Comment:
Removed the comment to fix highlighting and to avoid confusion (readers might get the impression that there are predefined selectors).

```suggestion
```json
"checks" : [
    {
        "selectorId": "agentsWithCPULarger60p",
        "condition": "= 0"
    }
]
```
```